packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -116,14 +125,25 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
             this._updateCaps(capabilities, 'app', app.app)
         }
 
+        /**
+         * evaluate buildIdentifier in case unique execution identifiers are present
+         * e.g., ${BUILD_NUMBER} and ${DATE_TIME}
+        */
+        try {
+            this._handleBuildIdentifier(capabilities)
```

Comment:
Why wrapping this in `try/catch`? I don't see `_handleBuildIdentifier` throwing any errors